nxviz/edges.py

```diff
@@ -34,10 +34,21 @@ def transparency(et: pd.DataFrame, alpha_by: Hashable):
     return pd.Series([0.1] * len(et), name="alpha")
 
 
-def edge_colors(et: pd.DataFrame, color_by: Hashable):
+def edge_colors(
+    et: pd.DataFrame,
+    nt: pd.Series,
```

Comment:
Strictly speaking, a node table should be an entire pandas DataFrame. 

```suggestion
    nt: pd.DataFrame,
```

Reply:
Oh, that's creepy mistake that I forgot to fix. Thank you for catching that.